libsolidity/interface/StandardCompiler.cpp

```diff
@@ -74,22 +74,22 @@ Json::Value formatError(
 
 Json::Value formatFatalError(string const& _type, string const& _message)
 {
-	Json::Value output = Json::objectValue;
+	Json::Value output{Json::objectValue};
 	output["errors"] = Json::arrayValue;
 	output["errors"].append(formatError(Error::Severity::Error, _type, "general", _message));
 	return output;
 }
 
 Json::Value formatSourceLocation(SourceLocation const* location)
 {
-	Json::Value sourceLocation;
-	if (location && location->sourceName)
-	{
-		sourceLocation["file"] = *location->sourceName;
-		sourceLocation["start"] = location->start;
-		sourceLocation["end"] = location->end;
-	}
+	if (!(location && location->sourceName))
```

Comment:
I think it reads better with less nesting.
```suggestion
	if (!location || !location->sourceName)
```